depthai_ros_msgs/CMakeLists.txt

```diff
@@ -18,6 +18,40 @@ add_message_files (
   Object.msg
   Objects.msg
   AutoFocusCtrl.msg
```

Comment:
Remove old messages?

Reply:
I will remove these old messages after this PR is nearly done, and the compatibility to the existing packages are met.